code/modules/mob/living/carbon/human/human_helpers.dm

```diff
@@ -110,6 +110,11 @@
 	var/obj/item/card/id/id = wear_id
 	if(HAS_TRAIT(src, TRAIT_UNKNOWN))
 		. = if_no_id //You get NOTHING, no id name, good day sir
+		var/list/identity = list(null, null, null)
+		SEND_SIGNAL(src, COMSIG_HUMAN_GET_VISIBLE_NAME, identity)
```

Comment:
Using the same signal in two places is usually no no. Make a new one for this case.